content-cache/src/nginx_manager.py

```diff
@@ -477,6 +477,7 @@ def _get_upstream_healthchecks_worker(upstream: str, config: LocationConfig) ->
     Returns:
         A string with the lua script for the healthcheck workers.
     """
+    valid_status_str = ",".join([str(status) for status in config.healthcheck_valid_status])
```

Comment:
`",".join(map(str, config.healthcheck_valid_status))`
`",".join((str(status) for status in config.healthcheck_valid_status))`